graylog2-web-interface/src/util/isLocalStorageReady.js

```diff
@@ -0,0 +1,17 @@
+const isLocalStorageReady = () => {
```

Comment:
This is not used and we can assume the localstorage will be available, otherwise the user cannot use the web interface.